src/components/Tree.js

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from 'react'
 import PropTypes from 'prop-types'
 import * as d3 from 'd3'
-import { parseNewick } from './Utils'
+import parseNewick from '../newick'
```

Comment:
Hmm. Should the newick file be moved into the components folder? That would be fine

Reply:
yes

Reply:
I think it would be better if all helper functions will also be available to user like
```js
import newickParser from 'phylo-react/parser'
const parsed = newickParser("<string>")
```

Reply:
do you think exposing the newick parser to the end user of phylo-react is useful? currently the parser is just used internally.

Reply:
It may help in some cases, as it convert data into easily interpretable representation.
Just in case when user doesn't want tree out of data, rather he/she want to the converted to hierarchical format.